pkg/helm/install.go

```diff
@@ -42,6 +42,11 @@ func (m *Mixin) Install() error {
 		return errors.Wrap(err, "couldn't get kubernetes client")
 	}
 
+	err = m.Init()
```

Comment:
This should be moved below line 58 so that don't execute init before we know that the input yaml is valid (this comment probably applies to all the actions).

Reply:
Nice catch; updated here and in the other actions.